generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity.service.ts

```diff
@@ -59,46 +60,62 @@ export class <%= entityClass %>Service {
         });
     }
 
-    query(req?: any): Observable<Response> {
-        let options: any = {};
+<%_ if(hasDate) { _%>
+    convertResponse(res: any): any {
```

Comment:
Good idea! We can have private methods in TypeScript - I always forget it :smile: